### PR TITLE
General Cleanup & Updates

### PR DESCRIPTION
**Pull request by xyzeva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -342,14 +339,27 @@ Jars are heuristically detected as Minecraft mods or plugins as follows:
...
+## Stage4 (`dummyloader3.jar`)
+This is a added on jar with the C2 server update which seems to be just the SkyRage updater, which is another minecraft malware targetting blackspigot.
+
+*This section needs more information, feel free to contribute*
+
+## New C2 (`107.189.3.101`)
+
+The new C2 came with a deobfuscated stage3, accidentialy pushed by the looks of it. After, the upload URI was changed to a Stage4, which is `SkyRage` the blackspigot malware.
+
+The new C2 is hosted on BuyVM, please make sure to report it if you know anyone working there.
```
Comment (jaskarth) [inline README.md:351]: Let's not do calls to action

File: README.md [hunk 2]
```diff
@@ -342,14 +339,27 @@ Jars are heuristically detected as Minecraft mods or plugins as follows:
...
+
 ## Other Stuff
 *The only official channel run by the same team that wrote this writeup*  is [#cfmalware on EsperNet IRC](https://webchat.esper.net/?channels=cfmalware) — we do not have a Discord. You may join the channel if you wish — due to an influx of new users we've set the channel +m, you will need permission to speak. **Joining an IRC channel will expose your IP address.**
 
-IRC logs: TODO
+IRC logs: ~~TODO~~ Done: [Discord #irc-logs](https://discord.gg/serverscanning), NOTE: This discord is unrelated, if you are here just to read logs please do not chat about the actual situation here.
```
Comment (jaskarth) [inline README.md:356]: Would prefer without

File: README.md [hunk 3]
```diff
@@ -381,6 +391,12 @@ Java edition modding has always had the full power of Java, and this is the othe
 Minecraft itself is not run with any sandboxing, and servers usually are not sandboxed unless the owner is knowledgeable enough to do so.
 
 Good sandboxing is difficult, especially on systems such as Linux where SELinux/AppArmor have such poor UX that no one deploys them.
+## Unmasking
+
```
Comment (jaskarth) [inline README.md:395]: Remove this section entirely

**Review by jaskarth — CHANGES_REQUESTED** [review]

**Comment by xyzeva** [comment]
GitHub seems to not be showing my new commit. Check my fork.

**Comment by jaskarth** [comment]
Strange, try force pushing or making a new PR maybe?

**Comment by xyzeva** [comment]
New PR Created #5

**Pull request closed without merging**